src/popup.js

```diff
@@ -0,0 +1,138 @@
+import { commentCounter } from './util.js';
+
+const commentAPI = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/gggCBSax83rR3Kv2Y5vr/';
+
+const mainSection = document.querySelector('#grid-layout');
+
+// function to close the description
+const closeDescription = () => {
+  const itemSection = document.querySelector('.item-section');
+  itemSection.remove();
+};
+
+const getComments = async (id, commentSection) => {
+  const response = await fetch(`${commentAPI}${'comments?item_id='}${id}`);
+  const data = await response.text();
+  const parsedData = JSON.parse(data);
+  const elemContainer = document.querySelector('.elem-container');
+  const counterDiv = document.createElement('div');
+  counterDiv.className = 'counter';
+  parsedData.forEach((comment) => {
+    const commentDiv = document.createElement('div');
+    commentDiv.classList.add('comment');
+    const commentName = document.createElement('h3');
+    const commentP = document.createElement('p');
+    commentName.innerText = comment.username;
+    commentP.innerText = comment.comment;
+    commentDiv.appendChild(commentName);
+    commentDiv.appendChild(commentP);
+    commentSection.appendChild(commentDiv);
+  });
+  counterDiv.innerText = `${commentCounter(parsedData)} ${'comments'}`;
+  elemContainer.appendChild(counterDiv);
+};
+// function to show description
+const showDescription = (e) => {
+  // gets the image from API
+  const itemImage = document.createElement('img');
+  const parentCard = e.target.parentElement;
+  // // create the item section
+
+  const itemName = document.createElement('h2');
+  itemName.innerText = e.target.parentElement.childNodes[3].textContent;
+
+  // // close button
+  const closeBtn = document.createElement('button');
+  closeBtn.className = 'fas fa-times';
+  closeBtn.addEventListener('click', closeDescription);
+
+  // // append the elements to the item section
+  const itemSection = document.createElement('div');
+  const elemContainer = document.createElement('div');
+  const nameInput = document.createElement('input');
+  const commentInput = document.createElement('input');
+
+  const submitBtn = document.createElement('button');
+
+  const commentSection = document.createElement('div');
+
+  const getImage = () => {
+    fetch(`https://pokeapi.co/api/v2/pokemon/${parentCard.childNodes[3].textContent}`)
+      .then((res) => res.json())
+      .then((data) => {
+        itemImage.src = data.sprites.front_default;
+        parentCard.id = data.id;
+        getComments(data.id, commentSection);
+      });
+  };
+  getImage();
```

Comment:
- [ ] Please make sure that your popup content shows details about the selected item. Currently, you are just showing an image. Add some extra details about the pokemon. 
💯 🥇